cdk/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "cdk",
-  "version": "0.0.0",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
```

Comment:
What's the purpose of the license field?

Reply:
It's required - but besides that - because we code in the open, we should have a license  indicating how people _could_ reuse the code. They probably won't.

It's also part of [our recommendations](https://github.com/guardian/recommendations/blob/654e08854c2c0d3e83616339f5558010830d8fbd/github.md?plain=1#L25) - I might try and add a "Why Apache 2" to the docs.